layouts/partials/docs/outdated_content.html

```diff
@@ -0,0 +1,20 @@
+{{ if ne .Site.Language.Lang "en" }}
+  {{ $currentLastmodUnixtime := .Lastmod.Unix }}
+  {{ $originCurrentLastmodUnixtime := 0 }}
+  {{ $originPageURL := "" }}
+  {{ $originPageTitle := "" }}
+  {{ range .Translations }}
+    {{ if eq .Lang "en" }}
+      {{ $originCurrentLastmodUnixtime = .Lastmod.Unix }}
+      {{ $originPageURL = .RelPermalink  }}
+      {{ $originPageTitle = .Title }}
+      {{ break }}
+    {{ end }}
+  {{ end }}
+
+  {{ if lt $currentLastmodUnixtime $originCurrentLastmodUnixtime }}
+  <div class="alert alert-danger caution callout" role="alert">
+    <strong>{{ T "outdated_content_title" }}:</strong> <p>{{ T "outdated_content_message" }} <a href="{{ $originPageURL }}">{{ $originPageTitle }}</a></p>
+  </div>
```

Comment:
```suggestion
  <div class="pageinfo pageinfo-primary outdated-localization-notice">
    <p><strong>{{ T "outdated_content_title" }}</strong></p>
    <p><span class="outdated-localization-notice-text">{{ T "outdated_content_message" }} <a href="{{ $originPageURL }}">{{ $originPageTitle }}</a></span></p>
  </div>
```
Try this?

Reply:
Thank you for your advice!
I adopted it :)

![image](https://github.com/kubernetes/website/assets/24271557/072da506-4a88-48c9-a568-e0438945b56e)